### ISSUE TITLE
Aibrowser for multimedia: introducing multimedia content accessibility for visually impaired users

### ISSUE THREAD

**Issue by Wotipati** [Wotipati/Blind_Accessibility_Papers#227]
### Links
- Paper: https://dl.acm.org/doi/10.1145/1296843.1296860

### Abstract
Multimedia content with Rich Internet Applications using Dynamic HTML (DHTML) and Adobe Flash is now becoming popular in various websites. However, visually impaired users cannot deal with such content due to audio interference with the speech from screen readers and intricate structures strongly optimized for sighted users.
We have been developing an Accessibility Internet Browser for Multimedia (aiBrowser) to address these problems. The browser has two novel features: non-visual multimedia audio controls and alternative user interfaces using external metadata. First, by using the aiBrowser, users can directly control the audio from the embedded media with fixed shortcut keys. Therefore, this allows blind users to increase or decrease the media volume, and pause or stop the media to handle conflicts between the audio of the media and the speech from the screen reader. Second, the aiBrowser can provide an alternative simplified user interface suitable for screen readers by using external metadata, which can even be applied to dynamic content such as DHTML and Flash.
In this paper, we discuss accessibility problems with multimedia content due to streaming media and the dynamic changes in such content, and explain how the aiBrowser addresses these problems by describing non-visual multimedia audio controls and external metadata-based alternative user interfaces. The evaluation of the aiBrowser was conducted by comparing it to JAWS, one of the most popular screen readers, on three well known multimedia-content-intensive websites.
The evaluation showed that the aiBrowser made the content that was inaccessible with JAWS relatively accessible by using the multimedia audio controls and alternative interfaces with metadata which included alternative text, heading information, and so on. It also drastically reduced the keystrokes for navigation with aiBrowser, which implies to improve the non-visual usability.

- ウェブブラウジングシステムを提案
- ショートカットキーを使いながら，スクリーンリーダーの挙動を制御する
